src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -735,6 +775,12 @@ class DistributedClosestPointImpl
     auto it = bvh->getTraverser();
     const int rank = m_rank;
 
+    double* sqDistThresh =
+      axom::allocate<double>(1, axom::execution_space<ExecSpace>::allocatorID());
+    sqDistThresh[0] = m_sqDistanceThreshold;
```

Comment:
... but perhaps this one should be 

```suggestion
    *sqDistThresh = m_sqDistanceThreshold;
```